### ISSUE TITLE
Random seeds are not working

### ISSUE THREAD

**Issue by jindrahelcl** [ufal/neuralmonkey#194]
I ran two experiments with nearly-identical configuration and same codebase. The only difference was in specifying `random_seed` in the second experiment's configuration. When I set the `random_seed` (differently from the default value) in the INI file, the results of both the experiments were the same.

For more details, you can look at these experiment directories:
```
~helcl/neuralmonkey/experiments/mmmt-hotel-november-alpha
~helcl/neuralmonkey/experiments/mmmt-hotel-november-bravo
```

**Comment by jindrahelcl** [comment]
změnils tam blbě to co se vrací z tý jedný funkce takže to teď neni args.věc, ale args.main["věc"], takže namespace můžeš zahodit, páč už to nedělá, co mělo - dřív se namespace dělal jen z mainu.

**Comment by obo** [comment]
To speed up this, I'd suggest removing randseed customization totally for now and have just the hardcoded number in the code directly. When you do so, please remove this issue from the milestone (and keep it open if it's not fully resolved).

**Comment by tomasmcz** [comment]
Should be fixed by e88da1512abb33fb54a4519340259426819f2e92 . Funny that it didn't break anything else.
